test/integrations/rewriteframes.test.ts

```diff
@@ -0,0 +1,579 @@
+jest.mock('../../src/utils/webViewUrl');
+
+import type { Exception } from '@sentry/browser';
+import { defaultStackParser, eventFromException } from '@sentry/browser';
+import type { Client, Event, EventHint, StackFrame, Stacktrace } from '@sentry/types';
+
+import { createCapacitorRewriteFrames } from '../../src/integrations/rewriteframes';
+import * as webViewUrl from '../../src/utils/webViewUrl';
+
+describe('RewriteFrames', () => {
```

Comment:
Can we add a dev and release iOS examples?

Reply:
the stacktrace for ios is the same from the tests so I'll skip it, but I added a test for dev builds